hnn_core/gui/gui.py

```diff
@@ -937,17 +937,23 @@ def _get_connectivity_widgets(conn_data):
 
     style = {'description_width': '150px'}
     style = {}
-    sliders = list()
+    conn_widgets = list()
     for receptor_name in conn_data.keys():
-        w_text_input = BoundedFloatText(
+        weight_text_input = BoundedFloatText(
             value=conn_data[receptor_name]['weight'], disabled=False,
             continuous_update=False, min=0, max=1e6, step=0.01,
             description="weight", style=style)
+        probability_text_input = BoundedFloatText(
+            value=conn_data[receptor_name]['probability'], disabled=False,
```

Comment:
A simple test to run would be to reduce the probability of one of the connections, and assert that there are fewer `gid_pairs`